benches/payout-stakers/src/main.rs

```diff
@@ -10,47 +12,105 @@ use aleph_client::{
     payout_stakers_and_assert_locked_balance, staking_batch_bond, staking_batch_nominate,
     staking_bond, staking_validate, wait_for_next_era, Connection,
 };
-use primitives::staking::{
-    MAX_NOMINATORS_REWARDED_PER_VALIDATOR, MIN_NOMINATOR_BOND, MIN_VALIDATOR_BOND,
+use primitives::{
+    staking::{MAX_NOMINATORS_REWARDED_PER_VALIDATOR, MIN_NOMINATOR_BOND, MIN_VALIDATOR_BOND},
+    TOKEN,
 };
 use sp_core::crypto::AccountId32;
 
 // testcase parameters
-const NOMINATOR_COUNT: u64 = MAX_NOMINATORS_REWARDED_PER_VALIDATOR as u64;
-const VALIDATOR_COUNT: u64 = 10;
-const ERAS_TO_WAIT: u64 = 10;
+const NOMINATOR_COUNT: u32 = MAX_NOMINATORS_REWARDED_PER_VALIDATOR;
+const ERAS_TO_WAIT: u32 = 100;
 
 // we need to schedule batches for limited call count, otherwise we'll exhaust a block max weight
 const BOND_CALL_BATCH_LIMIT: usize = 256;
-const NOMINATE_CALL_BATCH_LIMIT: usize = 192;
+const NOMINATE_CALL_BATCH_LIMIT: usize = 220;
 const TRANSFER_CALL_BATCH_LIMIT: usize = 1024;
 
+#[derive(Debug, Parser, Clone)]
+#[clap(version = "1.0")]
+struct Config {
+    /// WS endpoint address of the node to connect to. Use IP:port syntax, e.g. 127.0.0.1:9944
+    #[clap(long, default_value = "127.0.0.1:9944")]
+    pub address: String,
+
+    /// A path to a file that contains the root account seed.
+    /// If not given, Alice is assumed to be the root.
+    #[clap(long)]
+    pub root_seed_file: Option<String>,
+
+    /// A path to a file that contains seeds of validators, each in a separate line.
+    /// If not given, validators 0, 1, ..., validator_count are assumed
+    #[clap(long)]
+    pub validators_seed_file: Option<String>,
+
+    /// number of testcase validators
+    #[clap(long, default_value = "10")]
+    pub validator_count: u32,
```

Comment:
`validators_seed_file` and `validator_count` should be mutually exclusive

Reply:
Good point, done